dcp_stats/project_stats.py

```diff
@@ -0,0 +1,60 @@
+## Prod Project Rereival
+
+import requests, pprint, hca, csv, os
+
+azul_project_link = 'https://service.explore.data.humancellatlas.org/repository/projects?size=200'
+dss_client = hca.dss.DSSClient(swagger_url='https://dss.data.humancellatlas.org/v1/swagger.json')
+
+
+def get_bundles(project_uuid: str):
+    query = { "query": { "bool": { "must": [ { "match": { "files.project_json.provenance.document_id": project_uuid } } ] } } }
+    resp = dss_client.post_search(replica='aws', es_query=query)
+    assert resp.get('total_hits') != 0
+    bundles = [x.get('bundle_fqid') for x in resp.get('results')]
```

Comment:
Is it possible for some of the values in this list to be `None`?

If not, could we strictly enforce `x['bundle_fqid']` instead of `x.get('bundle_fqid')`.  Alternatively, if it can be `None`, could we remove those values, for example: `[x.get('bundle_fqid') for x in resp.get('results') if x.get('bundle_fqid')]`

Reply:
It would either be only `None`, or have valid bundle_fqids, ES should not be returning a mixture of both.